README.md

```diff
@@ -5,3 +5,6 @@
 | Dimensionality reduction | Víddakylfa |
 | Random forest | Slembiskógur |
 | Big data | Ginnungagögn |  
+| Regression | Aðhvarfsgreining |
+| Training data | Þjálfunargögn |    
+| Gradient Descent | Aðferð mesta bratta |  
```

Comment:
Spurning hvort að Gradient Descent sé betur lýst sem Rassaþotu?

Reply:
Aðferð mesta bratta er allavega allt of generic

Reply:
Gradient Descent with Momentum er þá Rassaþota með hlussu